airflow/www/static/js/datasets/Details.tsx

```diff
@@ -30,12 +30,74 @@ import {
   Table, TimeCell, CodeCell, TaskInstanceLink,
 } from 'src/components/Table';
 import { ClipboardButton } from 'src/components/Clipboard';
+import type { API } from 'src/types';
+import InfoTooltip from 'src/components/InfoTooltip';
 
 interface Props {
   datasetId: string;
   onBack: () => void;
 }
 
+const Details = ({
+  dataset: {
+    createdAt,
+    updatedAt,
+    uri,
+    extra,
+    upstreamTaskReferences,
+    downstreamDagReferences,
+  },
+}: { dataset: API.Dataset }) => (
+  <Box>
+    <Heading my={2} fontWeight="normal">
+      Dataset:
+      {' '}
+      {uri}
+      <ClipboardButton value={uri} iconOnly ml={2} />
+    </Heading>
+    {!!extra && (
+      <Flex>
+        <Text mr={1}>Extra:</Text>
+        <Code>{JSON.stringify(extra)}</Code>
+      </Flex>
+    )}
+    <Flex my={2}>
+      <Text mr={1}>Updated At:</Text>
+      <Time dateTime={updatedAt} />
+    </Flex>
+    <Flex my={2}>
+      <Text mr={1}>Created At:</Text>
+      <Time dateTime={createdAt} />
+    </Flex>
```

Comment:
```suggestion
```

In this context, adding updated and created at are maybe a bit confusing.  Cus "updated at" could indicated "the dataset was _updated_" i.e. in an event.

Reply:
Oh that isn't the case, its just when the db entry was updated? Ok. I'll cut that out.